xbmc/network/httprequesthandler/python/HTTPPythonWsgiInvoker.h

```diff
@@ -29,13 +29,14 @@ class CHTTPPythonWsgiInvoker : public CHTTPPythonInvoker
   CHTTPPythonWsgiInvoker(ILanguageInvocationHandler* invocationHandler, HTTPPythonRequest* request);
   ~CHTTPPythonWsgiInvoker() override;
 
+  static void globalInitializeModules(void);
```

Comment:
```suggestion
  static void GlobalInitializeModules();
```

ref: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#86-methods
and: https://github.com/xbmc/xbmc/blob/master/docs/CODE_GUIDELINES.md#36-superfluous-void

Reply:
@lrusak, you are right this updated in the last commit.